src/reducers/reducer.js

```diff
@@ -0,0 +1,19 @@
+const reducer = (state, action) => {
+    switch (action.type) {
+        case 'ADD':
```

Comment:
The common practice is to add action names into a separate "constants.js" file. 
E.g. `const ADD = 'ADD'`